HomeTask8/src/main/java/org/example/servlet/ServletStudentsList.java

```diff
@@ -0,0 +1,33 @@
+package org.example.servlet;
+
+import org.example.HibernateUtil;
+import org.example.entity.StudentHW8;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.query.Query;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+public class ServletStudentsList extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        getServletContext().getRequestDispatcher("/WEB-INF/show/searchStudents.html").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+        Session session = sessionFactory.openSession();
+
+        Query<StudentHW8> query = session.createQuery(" FROM StudentHW8 where groupId = " + req.getParameter("id"), StudentHW8.class);
```

Comment:
It's not a mistake, but, please, pay attention on it: it's better to avoid using 'generic' names, such as `id` here, because it's not obvious, which id we pass here. In such scenarios it' better to use `groupId` as a request parameter